### PR TITLE
webinars

### PR DESCRIPTION
**Pull request by ovcOS**

### CODE REVIEW

File: src/layouts/contentDiffusion.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  defaultImage,
+  external
+}: {
+  post: Page,
+  to: string,
+  defaultImage: Object,
+  external?: boolean
+}) => {
+  return (
+    <div className="card hover-shadow-5 bg-pale-secondary mb-5">
```
Comment (jahlela) [inline src/layouts/contentDiffusion.jsx]: NIT: I know this is consistent with the blog page, but these feel pretty cramped. What do you think about `mb-6` for both instead?

Screen Shot 2019-08-08 at 11 40 56 AM

File: src/layouts/index.jsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/no-array-index-key */
```
Comment (jahlela) [inline src/layouts/index.jsx]: What's the reasoning behind ignoring this rule? Generally indexes on arrays are a good idea

File: src/locale/fr/messages.po [hunk 3]
```diff
@@ -332,11 +332,15 @@ msgstr "Convertis le scénario final en seulement deux clics"
...
 
-#: src/pages/pricing.jsx:24
+#: src/layouts/index.jsx:124
+msgid "Convertible Loans"
+msgstr "Billet convertible"
+
+#: src/pages/pricing.jsx:25
 msgid "Convertibles"
-msgstr "Décapotables"
+msgstr "Billet convertible"
```
Comment (jahlela) [inline src/locale/fr/messages.po]: Is this meant to be unpluralized? as opposed to`Billets convertible`

File: src/locale/fr/messages.po [hunk 4]
```diff
@@ -332,11 +332,15 @@ msgstr "Convertis le scénario final en seulement deux clics"
...
 
-#: src/layouts/index.jsx:227
+#: src/layouts/index.jsx:141
 msgid "Cookie Policy"
 msgstr "Politique de Cookies"
```
Comment (jahlela) [inline src/locale/fr/messages.po]: Wow, cookie politics is hilarious

File: src/locale/fr/messages.po [hunk 5]
```diff
@@ -785,15 +797,19 @@ msgstr "En savoir plus sur la collaboration"
 msgid "Learn more about document management"
 msgstr "En savoir plus sur la gestion documentaire"
 
+#: src/pages/webinars.jsx:17
+msgid "Ledgy Webinars"
+msgstr "Webinaires Ledgy"
```
Comment (jahlela) [inline src/locale/fr/messages.po]: Wasn't the conclusion to keep it as `Webinars Ledgy?`

File: src/locale/fr/messages.po [hunk 6]
```diff
@@ -1433,6 +1453,15 @@ msgstr "Nous avons commencé à utiliser Ledgy tout en préparant une grande et
 msgid "We understand how sensitive equity information is for any business. That’s why your privacy and your data’s security is crucial to us. Here is what we do to protect your data."
 msgstr "Nous comprenons à quel point les informations relatives aux actions sont sensibles pour n’importe quelle entreprise. C’est pourquoi la protection de ta vie privée et la sécurité de tes données sont cruciales pour nous. Voici ce que nous faisons pour protéger tes données."
 
+#: src/layouts/index.jsx:109
+#: src/pages/webinars.jsx:13
+msgid "Webinars"
+msgstr "Webinaires"
```
Comment (jahlela) [inline src/locale/fr/messages.po]: Same here (and below) with les `webinaires` v `webinars`

File: src/locale/de/messages.po [hunk 7]
```diff
@@ -784,15 +796,19 @@ msgstr "Erfahre mehr über Kollaboration"
 msgid "Learn more about document management"
 msgstr "Erfahre mehr über Dokumentenverwaltung"
 
+#: src/pages/webinars.jsx:17
+msgid "Ledgy Webinars"
+msgstr "Ledgy Webinaire"
```
Comment (jahlela) [inline src/locale/de/messages.po]: In German?

**Comment by morloy** [comment]
image

* How about using screens from the actual video as banner images? The current ones are very dark.
* Are you planning to remove the URLs from the descriptions? Or make them at least look a bit nicers? 😉

File: src/layouts/contentDiffusion.jsx (hunk 1 above)
Comment (ovcOS) [reply src/layouts/contentDiffusion.jsx]: fixed the layout a bit, let me know if you think it's better or you'd still add more spacing

File: src/layouts/index.jsx (hunk 2 above)
Comment (ovcOS) [reply src/layouts/index.jsx]: If you include an array index in the `key` prop eslint will tell you it's bad for business. That is usually true for dynamic arrays, but in this case they are all static so it's safe to include indexes. In order not to include the `// disable ...` on each line, I applied it to the whole file

File: src/locale/fr/messages.po (hunk 4 above)
Comment (ovcOS) [reply src/locale/fr/messages.po]: 🤣

File: src/layouts/contentDiffusion.jsx (hunk 1 above)
Comment (jahlela) [reply src/layouts/contentDiffusion.jsx]: I don't see any change? But it's a small nit, anyway, so it's fine if you want to merge it.

File: src/layouts/contentDiffusion.jsx [hunk 8]
```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { Trans } from '@lingui/react';
+import { Link } from 'gatsby';
+import Img from 'gatsby-image';
+
+import { ChevronRight, targetBlank } from '../layouts/utils';
+
+export const ContentHeader = ({ heading }: { heading: string }) => (
```
Comment (morloy) [inline src/layouts/contentDiffusion.jsx]:
```suggestion
export const ContentHeader = ({ heading }: {| heading: string |}) => (
```
Can you make the types exact in all these components?

File: src/layouts/contentDiffusion.jsx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+  external?: boolean,
+  description: string | Node
+}) => {
+  return (
+    <div className="card hover-shadow-5 bg-pale-secondary mb-6">
+      <div className="row">
+        <div className="col-md-3">
+          {external ? (
+            <a href={to} {...targetBlank}>
+              <Img
```
Comment (morloy) [inline src/layouts/contentDiffusion.jsx]: I would put this image tag in a variable an not duplicate it.

File: src/layouts/contentDiffusion.jsx [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+                  <h5>{post.title}</h5>
+                </Link>
+              )}
+            </div>
+            <small className="col-md-2 text-md-right text-muted">{post.date}</small>
+            <div className="col-12">{description}</div>
+            <div className="col-12 mt-auto">
+              {external ? (
+                <a href={to} {...targetBlank}>
+                  <Trans>See video</Trans>
```
Comment (morloy) [inline src/layouts/contentDiffusion.jsx]:
```suggestion
                  <Trans>Watch now</Trans>
```

File: src/layouts/index.jsx (hunk 2 above)
Comment (morloy) [reply src/layouts/index.jsx]: I agree with @jahlela. And in this particular case, it looks like the `link` is already a pretty decent key. I would just use that, or something like ``key={`help-${link}`})``

File: src/pages/webinars.jsx [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+export const pageQuery = graphql`
+  fragment CoverImage on ImageSharp {
+    fluid(maxWidth: 200, maxHeight: 200, cropFocus: CENTER) {
+      ...GatsbyImageSharpFluid
+    }
+  }
+  query {
+    ledgy: imageSharp(fluid: { originalName: { regex: "/ledgy.png/" } }) {
+      ...CoverImage
+    }
```
Comment (morloy) [inline src/pages/webinars.jsx]: We have all these fragments defined in `blog.jsx`. Can we share them?

**Pull request merged**